pyvera/__init__.py

```diff
@@ -549,7 +549,6 @@ def set_service_value(
         value: Any,
     ) -> None:
         """Set a variable on the vera device.
-
```

Comment:
Please put back this blank line to maintain consistency with the rest of the code base.

Reply:
Done